RG_CHANGELOG.rst

```diff
@@ -0,0 +1,21 @@
+RG Changelog
+############
+
+All notable changes to this project will be documented in this file.
+
+The format is based on `Keep a Changelog <https://keepachangelog.com/en/1.0.0/>`_,
+and this project adheres to customized Semantic Versioning e.g.: `redwood-rg.1`
+
+[Unreleased]
+************
+
+2024-11-28
+****************************************
```

Comment:
[nit] We don't need to document the date of change because we can get it from git history

Reply:
I deleted the date, but I don’t understand why we need `RG_CHANGELOG` in this case?

Reply:
When we cut the release, we will have the date of release instead of `[Unreleased]` heading.
We will have the same changelog structure like in our branding repos (edx-theme, header, footer, etc.)